fineract-provider/build.gradle

```diff
@@ -87,8 +87,9 @@ dependencyManagement {
         dependency 'org.apache.commons:commons-email:1.5'
         dependency 'org.apache.commons:commons-io:+'
         dependency 'org.drizzle.jdbc:drizzle-jdbc:1.4'
-        dependency 'com.lowagie:itext:2.1.7'
-        dependency 'com.lowagie:itext-rtf:2.1.7'
+        dependency 'com.lowagie:itext:4.2.2'
+        dependency 'com.lowagie:itext-rtf:4.2.2'
+        dependency 'com.itextpdf:itextpdf:5.5.13'
```

Comment:
The mix of v4 and v5 here seems curious... is this really right?

Reply:
When I updated itext, I got error, requiring com.itextpdf:itextpdf. It seems that they are independent vendors. Version 5 is the most current version and dates back to 2011.

Reply:
https://mvnrepository.com/artifact/com.itextpdf/itextpdf